core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -53,6 +53,7 @@
  * @property integer $layout
  * @property boolean $includeCache
  * @property integer $cache
+ * @property integer $forceCache
```

Comment:
I guess this should be boolean?

Reply:
True fixed in 430dda656dcef832125c1ae6c0b7fa6a231e641d.